src/components/Pagination/Pagination.tsx

```diff
@@ -1,36 +1,70 @@
-import React from 'react';
+import React, { useState } from 'react';
 import cn from 'classnames';
 import s from './Pagination.module.scss';
 
-export const Pagination: React.FC = () => {
+type Props = {
+  selectOffset: number,
+  phonesLength: number,
+  selectLimit: number,
+  setSelectOffset: (offset: number) => void,
+};
+
+export const Pagination: React.FC<Props> = ({
+  selectOffset,
+  phonesLength,
+  selectLimit,
+  setSelectOffset,
+}) => {
+  const [currentPage] = useState(selectOffset);
+
+  const changeCountOfPages = (countOfItems: number) => {
```

Comment:
maybe better to give a name like `getPages`

Reply:
maybe. I`m fixing it anyway